.eslintrc.js

```diff
@@ -44,10 +44,19 @@ module.exports = {
         "indent": ["warn", 4, {"SwitchCase":1}],
         "linebreak-style": ["error", "unix"],
         "semi": ["error", "always"],
+        "keyword-spacing": ["error"],
+        "no-implicit-globals": ["error"],
+        "no-irregular-whitespace": ["error"],
         "no-new-object": ["error"],
-        "no-unneeded-ternary": ["error"],
+        "no-regex-spaces": ["error"],
+        "no-spaced-func": ["error"],
         "no-throw-literal": ["error"],
-        "no-implicit-globals": ["error"],
+        "no-unneeded-ternary": ["error"],
+        "no-whitespace-before-property": ["error"], // match flake8 E201 and E211
+        "space-before-function-paren": ["error", "never"],
+        "space-before-blocks": ["error"],
+        "space-in-parens": ["error", "never"],
+        "space-infix-ops": ["error"],   // match flake8 E225
```

Comment:
Along these lines, might want to add [key-spacing](https://eslint.org/docs/rules/key-spacing); the defaults seem good so I think we just want `"key-spacing": ["error"]`.